one-pager-maker/src/redux/document/documentType.ts

```diff
@@ -24,6 +26,9 @@ export type DocumentForCreate = Omit<Document, "id" | "deleted_at" | "updated_at
 export type DocumentForUpdate = Partial<Omit<Document, "updated_at">> & {
     id: string
 }
+export type Status = z.infer<typeof statusSchema>;
+export const statusValues = Object.values(statusSchema.enum);
+export type UrlPrivilege = z.infer<typeof privilegeSchema>;
```

Comment:
Is `urlPrivilegeValues` unnecessary?

Reply:
Thank you, yes, it's better to add.
I added it in latest commit.